src/runtime_src/core/tools/xbutil2/OO_HostMem.cpp

```diff
@@ -117,7 +117,7 @@ OO_HostMem::execute(const SubCmdOptions& _options) const
 
     // Exit if ENABLE action is specified but size is not
     if(enable && size == 0)
-      throw xrt_core::error(std::errc::invalid_argument, "Please specify a non-zero memory size.");
+      throw xrt_core::error(std::errc::invalid_argument, "Please specify a non-zero memory size between 4M and 1G.");
```

Comment:
Hmm..are there other check regarding the size of the memory.

In other words, what happens if one sets it to 1K, will there be an error produced?

Reply:
Yes, but that level of error handling is done by the driver. I had a chat with Chien-Wei about this earlier.